### PR TITLE
Fix a crash when alias metadata is attached to a call to lifetime inrinsic

### PR DESCRIPTION
**Pull request by MrSidims**
The correct translation of this case will be done later, once the spec
is updated.

Signed-off-by: Dmitry Sidorov <dmitry.sidorov@intel.com>

### CODE REVIEW

File: test/transcoding/SPV_INTEL_memory_access_aliasing/intel-alias-lifetime.ll [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+!opencl.ocl.version = !{!7}
+!opencl.used.extensions = !{!8}
+!opencl.used.optional.core.features = !{!8}
+!spirv.Generator = !{!9}
+
+!1 = !{i32 1, i32 1, i32 1}
+!2 = !{!"none", !"none", !"none"}
+!3 = !{!"int*", !"int*", !"int*"}
+!4 = !{!"", !"", !""}
+!5 = !{!"int*", !"int*", !"int*"}
```
Comment (mlychkov) [inline test/transcoding/SPV_INTEL_memory_access_aliasing/intel-alias-lifetime.ll:41]: They are not used in test code.
```suggestion
```

Comment (MrSidims) [reply test/transcoding/SPV_INTEL_memory_access_aliasing/intel-alias-lifetime.ll]: Lets go wild and remove even more metadata :)
Applied

**Review by mlychkov — APPROVED** [review]

**Comment by MrSidims** [comment]
Restart code style checks

**Comment by MrSidims** [comment]
@AlexeySotkin @AlexeySachkov @Fznamznon could you please take a look?

**Comment by MrSidims** [comment]
Ping @AlexeySotkin

File: test/transcoding/SPV_INTEL_memory_access_aliasing/intel-alias-lifetime.ll [hunk 2]
```diff
@@ -0,0 +1,30 @@
+; The test checks if the translator won't crash
+; TODO need to figure out how to translate the alias.scope/noalias metadata
+; in a case when it attached to a call to lifetime intrinsic
```
Comment (AlexeySotkin) [inline test/transcoding/SPV_INTEL_memory_access_aliasing/intel-alias-lifetime.ll:3]: How are they translated now?

Comment (MrSidims) [reply test/transcoding/SPV_INTEL_memory_access_aliasing/intel-alias-lifetime.ll:3]: lifetime start/stop are mapped on OpLifetimeStart/Stop. alias.scope/noalias metadata attached to these instructions are currently being dropped.

**Review by AlexeySotkin — APPROVED** [review]

**Review by AlexeySotkin — APPROVED** [review]

**Pull request merged**